### ISSUE TITLE
Tab decorations are not shown for right half when DPI > 1.0

### ISSUE THREAD

**Issue by Omrigan** [outfoxxed/hy3#48]
Hi!

Please, see the screenshot 
![screenshot-2023-11-29_20-04-13_584712837](https://github.com/outfoxxed/hy3/assets/2663743/3ad6f165-b899-46bb-a646-bac563c839fc)

This happens for the bottom half as well. What might be the problem?

**Comment by outfoxxed** [comment]
whats your monitor configuration? do you have multiple? nvidia?

**Comment by Omrigan** [comment]
> whats your monitor configuration? do you have multiple? nvidia?

Just default laptop eDP-1, only Intel iGPU. I started researching hy3 because of sway's poor support for Nvidia reverse PRIME (I have external connectors wired into dGPU), but this bug reproduces with Nvidia disabled.

I narrowed it down! It happens because of increased DPI. If I set it to 2, then the right half misses the tab decorations. If I set it to e.g., then it works for the two columns, but if I open more, like 3 or 4, then the tabs disappear on the rightmost column. 

I hope it helps!
